### PR TITLE
ch footer and mm

### PR DESCRIPTION
**Pull request by lstankus**

### CODE REVIEW

File: hawkshead/scss/partials/mega_menu.scss [hunk 1]
```diff
@@ -49,10 +49,10 @@
     padding-bottom: 0.5%;
 }
 
-.s-main-menu .mm__container .mm__column--collections .columnHead a,
+/*.s-main-menu .mm__container .mm__column--collections .columnHead a,
 .s-main-menu .block .category-items a {
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: delete commented out styles

File: hawkshead/scss/partials/mega_menu.scss [hunk 2]
```diff
@@ -383,4 +383,315 @@ ul.mm__column.mm__column--img.expanded {
...
+    width: 33.33% !important;
+}
+.column51{
+    width: 49% !important;
+}
+
+a.collection-moutdoor,
+a.collection-mactive,
+a.collection-minsect,
+a.collection-mcycle,
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: use 1 class instead of individual classes

File: hawkshead/scss/partials/mega_menu.scss [hunk 3]
```diff
@@ -383,4 +383,315 @@ ul.mm__column.mm__column--img.expanded {
...
+.s-main-menu .categories > li .mmColumn .collectionItem a:hover{
+  color:#fff;
+}
+
+@media only screen and (max-width:1227px) {
+  .collectionItem {
+    max-width: 180px;
+  }
+  .s-main-menu .categories > li .mmColumn .collectionItem a.collection-moutdoor,
+  .s-main-menu .categories > li .mmColumn .collectionItem a.collection-mdofe,
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: same as above

File: hawkshead/scss/partials/mega_menu.scss [hunk 4]
```diff
@@ -383,4 +383,315 @@ ul.mm__column.mm__column--img.expanded {
...
+  }
+
+  .collection-title-vertical {
+    text-align: left;
+    padding: 0;
+    color: #000;
+  }
+
+  .s-main-menu .block .category-items .mmColumn,
+  .s-main-menu .block .category-items .mmColumn {
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: same selector

File: hawkshead/scss/partials/mega_menu.scss [hunk 5]
```diff
@@ -383,4 +383,315 @@ ul.mm__column.mm__column--img.expanded {
...
+
+  .s-main-menu .categories > li .mmColumn .collectionItem a {
+    color: #343334;
+    text-align: left;
+  }
+
+  .s-main-menu .categories > li .mmColumn .collectionItem a,
+  .s-main-menu .categories > li .mmColumn .collectionItem a.collection-moutdoor,
+  .s-main-menu .categories > li .mmColumn .collectionItem a.collection-mdofe,
+  .s-main-menu .categories > li .mmColumn .collectionItem a.collection-mdiscovery,
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: same as other classes comment above

File: hawkshead/scss/partials/mega_menu.scss [hunk 6]
```diff
@@ -383,4 +383,315 @@ ul.mm__column.mm__column--img.expanded {
...
+  .s-main-menu .categories > li .mmColumn .collectionItem a:hover{
+    color:#343334;
+  }
+
+}
+
+@media only screen and (max-width:700px) {
+  .s-main-menu .block .category-items .mmColumn,
+  .s-main-menu .block .category-items .mmColumn,
+  .s-main-menu .block .category-items .mmColumn.techColumn {
```
Comment (Zac-Rimmer) [inline hawkshead/scss/partials/mega_menu.scss]: same selector used twice

**Comment by lstankus** [comment]
changes made

**Pull request merged**